mne/gui/_viewer.py

```diff
@@ -291,7 +284,7 @@ def clear(self):  # noqa: D102
     @on_trait_change('scene.activated')
     def plot(self):
         """Add the points to the mayavi pipeline"""
-        _scale = self.scene.camera.parallel_scale if not _testing_mode() else 1
+        _scale = self.scene.camera.parallel_scale
```

Comment:
Travis seems to fail even on tests that don't even install mayavi, and it fails even though according to the log all tests passed, what does that mean?